docs/tutorial/11_token-UI.md

```diff
@@ -0,0 +1,459 @@
+# 11. Token User Interface
```

Comment:
I think I'm getting something wrong. Is this file the new version of writing-the-components?

Reply:
Kind of. This is where I show how to do the three actions that are needed for the UI:

1. Read information from the blockchain
2. Send transactions for the blockchain
3. Listen to blockchain events